app/controllers/members_controller.rb

```diff
@@ -7,8 +7,17 @@ def index
     end
 
     def show
-        @member = Member.find(params[:id])
-        render json: @member
+        @member = Member.where("room_id= ?",params[:room_id])
+        returnList = []
+        @member.find_each do |m|
+            temp = {}
+            temp.store("user_id",m[:user_id])
+            temp.store("user_name", User.where(id: m[:user_id]).first[:username]) 
```

Comment:
Change User.where to User.find( m[:user_id]). Only 1 user per user_id

Reply:
Still need to use .first because .find can also return an array. Will change later if cleaner way found.

Reply:
.find() returns only 1 object if 1 id is given
https://apidock.com/rails/ActiveRecord/Base/find/class